frontend/src/components/Modals/LargeModalHeader.tsx

```diff
@@ -1,5 +1,6 @@
 import { BookingType } from "@/types";
-import { getIconByBookingType } from "../ConsultantRow";
+
+import { getIconByBookingType } from "@/components/Staffing/helpers/utils";
```

Comment:
Var det ikke du som nektet å bruke mappe-navn som "helpers" og "utils" to begin with? 😄  hihi